content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -37,8 +37,7 @@ Refer to the following table for Windows operating system support in Kubernetes.
 
 | Kubernetes version | Windows Server LTSC releases | Windows Server SAC releases |
 | --- | --- | --- | --- |
-| *Kubernetes v1.17* | Windows Server 2019 | Windows Server ver 1809 |
-| *Kubernetes v1.18* | Windows Server 2019 | Windows Server ver 1809, Windows Server ver 1903, Windows Server ver 1909 |
```

Comment:
I think this is slightly different than our previous intent, though I agree with this direction. What we are saying here is that we will support an SAC release up to the time that Microsoft drops the support even if the k8s version is still supported, and if the user wants the support they will will need to upgrade the SAC. 

This sounds reasonable since if there is a bug with the OS it wouldn't be supported, but we should add a comment here that calls out this specifically so users can be aware when deciding between LTSC and SAC that if they stay on an older version k8s to continue getting support they will need to ready and willing to upgrade SAC releases as time goes on (which includes rebuilding container images). We would also need to ensure we have tests that cover the latest SACs against the releases.